### PR TITLE
Feat/prismic transform updates

### PR DESCRIPTION
**Pull request by melanierogan**
## Who is this for?

This covers the extra points we needed for ResultList in stores, events and exhibitions. 
See ticket https://github.com/wellcomecollection/wellcomecollection.org/issues/8825

## What is it doing for them?

We now have a valid link for the particular type, e.g. stories, i.e. you can click through to the article you found. 
Summary now comes from body - promo - captions, as per conversation with Danny.
We have labels pulling through, with the exception of serials until 'Part of' logic is worked out. 
We have contributors in the comma seperated format we wanted. 
The ResultList now has totalResults and indicators for previous and next, with a cursor pointer in the graphql query to pave the way for pagination. 
Where possible, I've implemented these in the FE (just for stories).

Screenshot 2022-11-10 at 16 38 39

Screenshot 2022-11-10 at 16 38 52

### CODE REVIEW

File: catalogue/webapp/pages/search/stories.tsx [hunk 1]
```diff
@@ -144,39 +144,37 @@ export const SearchPage: NextPageWithLayout<Props> = ({
...
-                          <LabelsList labels={[{ text: 'Article' }]} />
+                          <LabelsList labels={[story.label]} />
                         </MobileLabel>
                       )}
                     </ImageWrapper>
                     <Details>
-                      {story.type && (
+                      {story.label && (
                         <DesktopLabel>
                           {/* TODO add labels once we have them */}
```
Comment (rcantin-w) [inline catalogue/webapp/pages/search/stories.tsx:173]: Should we remove the TODO?

File: catalogue/webapp/pages/search/stories.tsx [hunk 2]
```diff
@@ -144,39 +144,37 @@ export const SearchPage: NextPageWithLayout<Props> = ({
...
                 <Container key={story.id}>
-                  {/* TODO add link once we have it */}
-                  <StoryWrapper href="#">
+                  <StoryWrapper href={story.url}>
                     <ImageWrapper>
                       <img src={story.image.url} alt="" />
 
                       {story.type && (
                         <MobileLabel>
                           {/* TODO add labels once we have them */}
```
Comment (rcantin-w) [inline catalogue/webapp/pages/search/stories.tsx:165]: Here too

File: catalogue/webapp/pages/search/stories.tsx (hunk 1 above)
Comment (melanierogan) [reply catalogue/webapp/pages/search/stories.tsx:173]: Eek, yes indeed

File: catalogue/webapp/services/prismic/types/exhibition.ts [hunk 3]
```diff
@@ -1,17 +1,22 @@
...
   firstPublicationDate: Date;
-  title: Title;
+  contributors: string[];
   type: ContentType[];
+  summary: Standfirst;
+  label: Label;
+};
+
+export type Label = {
+  text: string | '';
```
Comment (rcantin-w) [inline catalogue/webapp/services/prismic/types/exhibition.ts:19]: would just `text: string` work?

**Review by rcantin-w — APPROVED** [review]
🎉

**Pull request merged**